src/app/pages/submission-pages/sample-group-page/sample-group-page.component.ts

```diff
@@ -0,0 +1,560 @@
+import { PageService } from './../../../services/page.service';
```

Comment:
i can see that some of the code here is commented out because backend components i.e. data models arent ready yet. But its not clear why other some of the other code is also commented out. Can we put add some comments next to them explaining why?

Reply:
Does not really matters. I would refactor it anyway. As I said this is a draft code, not final at all. I need this code to be push to dev for the UI demo.

Reply:
oh ok. i'd say why bother with code reviews then 😉